.stylelintrc

```diff
@@ -7,10 +7,7 @@
     "scss/dollar-variable-pattern": null,
     "scss/dollar-variable-colon-space-after": null,
     "scss/at-mixin-pattern": null,
```

Comment:
Stylelint deprecated a number of rules in v15 that overlapped with Prettier:
https://stylelint.io/migration-guide/to-15#deprecated-stylistic-rules